run-bazel-in-docker.sh

```diff
@@ -0,0 +1,11 @@
+mkdir -p /tmp/build_output/ 
```

Comment:
did you mean to check in this file? I guess it's for testing something. If so, maybe add some comments here about what it's used for and how to use it

Reply:
Added some docs... I created this script because the repo's tests fail on non-linux systems (I'm on MacOS):
```
tests/container/nodejs/nodejs_image_custom_binary_test.image: line 208: /usr/bin/docker: No such file or directory
```